python_bindings/src/PyFunc.cpp

```diff
@@ -58,10 +58,26 @@ void define_func(py::module &m) {
         .def(py::init<>())
     ;
 
+    // Deliberately not supported, because they don't seem to make sense for Python:
+    // - set_custom_allocator()
+    // - set_custom_do_task()
+    // - set_custom_do_par_for()
+    // - jit_handlers()
+    // - add_custom_lowering_pass()
+    // - clear_custom_lowering_passes()
+    // - custom_lowering_passes()
+
+    // Not supported yet, because we want to think about how to expose runtime
```

Comment:
Is there an issue filed? If so, might be worth referencing it here. Otherwise, perhaps just a TODO comment.

Reply:
https://github.com/halide/Halide/issues/2790